generators/Exercises/Sublist.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Generators.Input;
+using Generators.Output;
+using Humanizer;
+using System.Linq;
+
+namespace Generators.Exercises
+{
+    public class Sublist : GeneratorExercise
+    {
+        protected override HashSet<string> AddAdditionalNamespaces() => new HashSet<string>() { typeof(System.Collections.Generic.IList<int>).Namespace };
```

Comment:
It looks like you've already included the `System.Collections.Generic` namespace so `typeof(IList<int>).Namespace` should work here.